app/src/lib/git/worktree.ts

```diff
@@ -110,38 +135,13 @@ async function findTemporaryWorkTrees(repository: Repository) {
   })
 }
 
-/**
- * Finds an existing temporary work tree for use in Desktop or creates a new one.
- * Won't modify the repository's working directory.
- * _The returned worktree will be checked out to the given commit._
- */
-export async function findOrCreateTemporaryWorkTree(
```

Comment:
since we no longer want to reuse worktrees automatically, this method is no longer needed.